src/main/java/com/gojek/beast/sink/bq/handler/impl/OOBErrorHandler.java

```diff
@@ -28,17 +30,22 @@ public Status handleErrorRecords(Map<Record, List<BQInsertionRecordsErrorType>>
         }
         boolean shouldBatchFail = false;
         final List<Record> recordsToWrite = new ArrayList<>();
+        int invalidRecordsCount = 0;
+        int unKnownErrorRecordsCount = 0;
         for (Record record : records.keySet()) {
             final List<BQInsertionRecordsErrorType> errorTypesForRecord = records.get(record);
             boolean recordHasOutOfBoundsData = false;
-            boolean recordHasInvalidData = false;
             for (final BQInsertionRecordsErrorType errorType : errorTypesForRecord) {
                 switch (errorType) {
                     case OOB: recordHasOutOfBoundsData = true;
```

Comment:
also add a metric for count of OOB messages

Reply:
Metrics are in GCSErrorWriter.